ports/servoshell/egl/ohos.rs

```diff
@@ -167,7 +167,10 @@ impl ServoAction {
     fn do_action(&self, servo: &Rc<RunningAppState>) {
         use ServoAction::*;
         match self {
-            WakeUp => servo.perform_updates(),
+            WakeUp => {
+                servo.perform_updates();
+                servo.present_if_needed();
```

Comment:
Hmm, I don't quite understand why this fixes anything. We would call present if needed on vsync anyway, which shouldn't be far off and i think don't stop